app/controllers/api/v1/trails_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::TrailsController < ApplicationController
+  def index
+    render json: TrailsSerializer.new(TrailsList.new(params[:location]))
+  end
+end
```

Comment:
i feel like there is a better way, as i'm using the poro model `TrailsList` as a search poro. it feels like i should be getting the info first and then passing it to the model, but... this was the fastest and most direct way i could think of. i wouldn't mind sitting down and discussing a better way.